CMakeLists.txt

```diff
@@ -113,8 +113,11 @@ set(CMAKE_INSTALL_DATAROOTDIR ${CMAKE_BINARY_DIR}/Root/res)
 
 if (${CMAKE_HOST_SYSTEM_NAME} MATCHES SerenityOS)
     message("Good job on building cmake!")
+    set(TOOLCHAIN_VERSION 10.2.0)
 else()
-    set(TOOLCHAIN_PATH ${CMAKE_SOURCE_DIR}/Toolchain/Local/${SERENITY_ARCH}/bin)
+    set(TOOLCHAIN_VERSION 10.3.0)
```

Comment:
Is there a reason it's 10.2.0 with CMake?

Reply:
Because that tracks the gcc port version, I think it's currently wrong as the port was later updated... but that's a separate issue.